src/components/Tag/Tag.jsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import _ from 'lodash';
+import { lucidClassNames } from '../../util/style-helpers';
+import CrossIcon from '../Icon/CrossIcon/CrossIcon';
+import {
+	createClass,
+	filterTypes,
+	rejectTypes,
+	omitProps,
+} from '../../util/component-types';
+
+const cx = lucidClassNames.bind('&-Tag');
+
+const {
+	bool,
+	func,
+	node,
+	string,
+	object,
+} = React.PropTypes;
+
+/**
+ * {"categories": ["communication"]}
+ *
+ * Organizes items into removable items or groups. Can be nested to facilitate
+ * grouping.
+ */
+const Tag = createClass({
+	displayName: 'Tag',
+
+	propTypes: {
+		/**
+		 * Shows or hides the little "x" for a given tag.
+		 */
+		isRemovable: bool,
+		/**
+		 * Called when the user clicks to remove a tag.
+		 *
+		 * Signature: `({props, event}) => {}`
+		 */
+		onRemove: func,
+		/**
+		 * Can contain elements or nested `Tag` components.
+		 */
+		children: node,
+		/**
+		 * Appended to the component-specific class names set on the root element.
+		 */
+		className: string,
+		/**
+		 * Styles that are passed through to the root container.
+		 */
+		style: object,
+	},
+
+	getDefaultProps() {
+		return {
+			onRemove: _.noop,
+		};
+	},
+
+	handleRemove(event) {
+		this.props.onRemove({ props: this.props, event });
+	},
+
+	render() {
+		const {
+			isRemovable,
+			children,
+			className,
+			style,
+			...passThroughs
+		} = this.props;
+
+		const subTags = filterTypes(children, Tag);
+		const otherChildren = rejectTypes(children, Tag);
+		const isLeaf = _.isEmpty(subTags);
+
+		return (
+			<div
+				{...omitProps(passThroughs, Tag)}
+				className={cx('&', { '&-leaf': isLeaf }, className)}
+				style={style}
```

Comment:
`style` can probably just be included in `...passThroughs` since it's not modified.